### ISSUE TITLE
[FEATURE_REQUEST] Sitemap generation tool

### ISSUE THREAD

**Issue by h4ppyr0gu3** [Lissy93/dashy#859]
### Is your feature request related to a problem? If so, please describe.

I am trying to publish to google and would have to generate a sitemap for all my subdomains, it would be nice if we could have a way to generate the sitemap just with the urls 

### Describe the solution you'd like

an intuitive way to configure a sitemap would be great

### Priority

Low (Nice-to-have)

### Is this something you would be keen to implement

No

**Comment by Lissy93** [comment]
Heya @h4ppyr0gu3 
Sorry for the late reply, it's been a busy couple of weeks.

For sitemap content, do you mean a list of all the URLS of apps added to your dashboard?

**Comment by h4ppyr0gu3** [comment]
yes, i am not too sure how it  should look but i think for indexing it on google it would be nice to have a generated sitemap which will just be a list of urls

**Comment by h4ppyr0gu3** [comment]
Love the product by the way, so thank you for you time and work on it
This feature request is just a thought
